.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
```

Comment:
No need to specifically point out that it is a request, feature/bug/or something else are belong to request,